src/coreclr/jit/importer.cpp

```diff
@@ -4235,6 +4235,24 @@ GenTree* Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolvedT
             break;
         }
 
+        case CORINFO_FIELD_STATIC_DATASEGMENT:
+        {
+#ifdef FEATURE_READYTORUN
+            assert(opts.IsReadyToRun());
+            assert((pFieldInfo->fieldFlags & CORINFO_FLG_FIELD_INITCLASS) == 0);
+            assert(pFieldInfo->fieldLookup.accessType == InfoAccessType::IAT_VALUE);
+            assert(fieldKind == FieldSeq::FieldKind::SimpleStatic);
+
+            GenTree*  baseAddr = gtNewIconHandleNode((size_t)pFieldInfo->fieldLookup.addr, GTF_ICON_STATIC_HDL);
+            FieldSeq* fseq     = GetFieldSeqStore()->Create(pResolvedToken->hField, pFieldInfo->offset, fieldKind);
```

Comment:
Why does `innerFldSeq` defined above not here?

Reply:
Good point, switch to innerFldSeq